internal/command/jsonformat/change/sensitive.go

```diff
@@ -0,0 +1,48 @@
+package change
+
+import (
+	"fmt"
+	"reflect"
+)
+
+func Sensitive(before, after interface{}, beforeSensitive, afterSensitive bool) Renderer {
+	return sensitiveRenderer{
+		before:          before,
+		after:           after,
+		beforeSensitive: beforeSensitive,
+		afterSensitive:  afterSensitive,
+	}
+}
+
+type sensitiveRenderer struct {
+	before interface{}
+	after  interface{}
+
+	beforeSensitive bool
+	afterSensitive  bool
+}
+
+func (renderer sensitiveRenderer) Render(change Change, indent int, opts RenderOpts) string {
+	return fmt.Sprintf("(sensitive)%s%s", change.nullSuffix(opts.overrideNullSuffix), change.forcesReplacement())
+}
+
+func (renderer sensitiveRenderer) Warnings(change Change, indent int) []string {
+	if (renderer.beforeSensitive == renderer.afterSensitive) || renderer.before == nil || renderer.after == nil {
+		// Only display warnings for sensitive values if they are changing from
+		// being sensitive or to being sensitive or if they are not being
+		// destroyed or created.
+		return []string{}
+	}
+
+	var warning string
+	if renderer.beforeSensitive {
+		warning = fmt.Sprintf("  # [yellow]Warning[reset]: this attribute value will no longer be marked as sensitive\n%s  # after applying this change.", change.indent(indent))
+	} else {
+		warning = fmt.Sprintf("  # [yellow]Warning[reset]: this attribute value will be marked as sensitive and will not\n%s  # display in UI output after applying this change.", change.indent(indent))
```

Comment:
The manual line breaks here may have been copied from the existing plan renderer, but jumped out at me as a source of ugly rendering at variable terminal widths. It'd be great if the renderer had support for a render width and line wrapping, including lines which are prefixed with comment characters.

Reply:
I've added that down into my list for future enhancements - I definitely think it is possible. I won't do it now then, as keeping the actual diff changes between the old and new one to a minimum makes my testing much easier.

Once I have it to par with the existing one, I think we can start implementing improvements.